extensions/machine-learning-services/src/common/constants.ts

```diff
@@ -62,12 +64,36 @@ export const rConfigError = localize('mls.rConfigError', "R executable is not co
 export const installingDependencies = localize('mls.installingDependencies', "Installing dependencies ...");
 export const resourceNotFoundError = localize('mls.resourceNotFound', "Could not find the specified resource");
 export const latestVersion = localize('mls.latestVersion', "Latest");
+export const localhost = 'localhost';
 export function httpGetRequestError(code: number, message: string): string {
 	return localize('mls.httpGetRequestError', "Package info request failed with error: {0} {1}",
 		code,
 		message);
 }
-
+export function getErrorMessage(error: Error): string { return localize('azure.resource.error', "Error: {0}", error?.message); }
+export const extLangInstallTabTitle = localize('extLang.installTabTitle', "Installed");
+export const extLangLanguageCreatedDate = localize('extLang.languageCreatedDate', "Installed");
+export const extLangLanguagePlatform = localize('extLang.languagePlatform', "Platform");
```

Comment:
Wrong text?

Reply:
sorry not sure what you mean, Platform seems to be correct 


---
In reply to: [375353736](https://github.com/microsoft/azuredatastudio/pull/9043#discussion_r375353736) [](ancestors = 375353736)